test/NuGet.Core.FuncTests/NuGet.Packaging.FuncTest/SigningTests/SignatureTrustAndValidityVerificationProviderTests.cs

```diff
@@ -102,11 +103,14 @@ public SignatureTrustAndValidityVerificationProviderTests(SigningTestFixture fix
             var timestampService = await _testFixture.GetDefaultTrustedTimestampServiceAsync();
             var keyPair = SigningTestUtility.GenerateKeyPair(publicKeyLength: 2048);
             var subjectName = new X509Name("CN=NuGet Test Expired Certificate");
-            var bcCertificate = ca.IssueCertificate(
-                keyPair.Public,
-                subjectName,
-                notBefore: DateTime.UtcNow.AddSeconds(-2), // Must be before the timestamp's lower limit
-                notAfter: DateTime.UtcNow.AddSeconds(10));
+            var now = DateTimeOffset.UtcNow;
+            var issueOptions = new IssueCertificateOptions(keyPair.Public)
+                {
+                    NotAfter = now.AddSeconds(10),
+                    NotBefore = now.AddSeconds(-2),
+                    SubjectName = new X509Name("CN=NuGet Test Expired Certificate")
```

Comment:
you could use the variable `subjectName` defined in line `106` instead of creating a new `X509Name`, or remove that unused variable and just create the subject name here.